manager/controllers/app/policy_compiler_interface.go

```diff
@@ -30,10 +40,44 @@ func ConstructOpenAPIReq(datasetID string, input *app.FybrikApplication, operati
 		Action: *operation,
 		Resource: policymanager.Resource{
 			ID: taxonomy.AssetID(datasetID),
+			Metadata: &datacatalog.ResourceMetadata{
+				Tags: taxonomy.Tags{Properties: serde.Properties{Items: map[string]interface{}{}}},
+			},
 		},
 	}
 }
 
+func ValidatePolicyDecisionsResponse(response *policymanager.GetPolicyDecisionsResponse, taxonomyFile string) error {
+	var allErrs []*field.Error
+
+	// Convert GetAssetRequest Go struct to JSON
+	responseJSON, err := json.Marshal(response)
+	if err != nil {
+		return err
+	}
+	log.Println("responseJSON (policy decisions):" + string(responseJSON))
+
+	// Validate Fybrik module against taxonomy
+	allErrs, err = validate.TaxonomyCheck(responseJSON, taxonomyFile)
+	log.Println("allErrs:")
+	log.Println(allErrs)
+	if err != nil {
+		log.Println("err: ")
```

Comment:
Use zerolog.Logger instead of log as in other manager files.

Reply:
fixed. Please check. Thanks.